charts/tempo-distributed/Chart.yaml

```
@@ -2,7 +2,7 @@ apiVersion: v2
 name: tempo-distributed
 description: Grafana Tempo in MicroService mode
 type: application
-version: 1.0.1
+version: 1.0.2
```

feature should bump to minor version.
```suggestion
version: 1.1.0
```

---

Thanks for the catch! Change has been made :+1: